tool/colors/colors_main_cupertino.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'flutter/colors_cupertino.dart';
+import 'stubs.dart';
+
+void main() {
+  // https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/cupertino/colors.dart
+
+  StringBuffer buf = new StringBuffer();
+  buf.writeln('# Generated file - do not edit.');
+  buf.writeln();
+  buf.writeln('# suppress inspection "UnusedProperty" for whole file');
+  buf.writeln();
+
+  // colors
+  final Map<String, Color> colors = <String, Color>{
```

Comment:
This is a lot of code that could potential get out of date wrt CupertinoColors and or will need maintenance.

To speak to your comment in the PR, could we reduce the amount of code here if we generate the properties files from a Flutter app, ala the current icons code?

If I remember correctly, the code to match colors was written before the code to generate icons, so just may not have been aware of the other way to generate these files.